db/interface.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding:utf-8 -*-
+
+from db.models import Payment, Detail
+
+
+class PaymentService(object):
+    """
+    Serves as an interface which takes requests from the front-end
+    and runs the corresponding actions in the database.
+    """
+
+    def __init__(self):
```

Comment:
sure. makes sense. once #98 , #101 , and #102 are merged in, i'll adjust the project structure to reflect this.